test/e2e/create_test.go

```diff
@@ -332,7 +332,8 @@ var _ = Describe("Podman create", func() {
 	})
 
 	It("podman create --authfile with nonexistent authfile", func() {
-		session := podmanTest.Podman([]string{"create", "--authfile", "/tmp/nonexistent", "--name=foo", ALPINE})
+		nonExistentFile := cachePath("nonexistent")
+		session := podmanTest.Podman([]string{"create", "--authfile", nonExistentFile, "--name=foo", ALPINE})
```

Comment:
I think this, and the other `/tmp/nonexistent` changes, are overkill.

Reply:
I reverted `cachePath("noexistent")` to `/tmp/noexistent`.